app/models/assignment.rb

```diff
@@ -13,23 +13,40 @@ class Assignment < ActiveRecord::Base
   validates :mixpanel_result, inclusion: { in: %w(success failure) }, allow_nil: true
   validate :variant_must_exist
 
-  scope :unsynced_to_mixpanel, -> { where("mixpanel_result = 'failure' OR mixpanel_result IS NULL") }
-
   normalize_attributes :mixpanel_result
 
   delegate :name, to: :split, prefix: true
 
-  class << self
-    def to_hash
-      Hash[all.includes(:split).map { |a| [a.split.name.to_sym, a.variant.to_sym] }]
-    end
+  scope :unsynced_to_mixpanel, -> { where("mixpanel_result = 'failure' OR mixpanel_result IS NULL") }
```

Comment:
I don't think this gets used anymore, does it? It feeds all the way back up to `unsynced_splits`, which I don't see any non-spec references to.

Reply:
k, i'll double check and kill it if so.

Reply:
kilt